.github/workflows/container-deploy.yml

```diff
@@ -74,4 +83,4 @@ jobs:
           # sudo docker exec -it docker_mongodb_1 -c "cd /data/db && tar -xzvf openfoodfacts-mongodbdump.tar.gz && mongorestore"
 
           # Import sample data into docker-compose db
-          sudo docker exec -it docker_backend_1 -c "bash /opt/scripts/import_sample_data.sh"
+          sudo docker-compose -f ./docker-compose.yml -f ./docker-compose.dev.yml -f ./docker-compose.deploy-test.yml exec backend bash /opt/scripts/import_sample_data.sh
```

Comment:
what's the difference between this and the previous command ?

Reply:
With `docker exec` you need to assume the actual container name (ie. `docker_` prefix and `_1` suffix etc.) while `docker-compose` knows the service name through the provided compose files, and can simply execute the command in a service it knows to be running.